app/views/layouts/mission_control/jobs/application.html.erb

```diff
@@ -9,7 +9,7 @@
   <meta name="turbo-cache-control" content="no-cache">
   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@1.0.1/css/bulma.min.css">
   <%= stylesheet_link_tag "mission_control/jobs/application", "data-turbo-track": "reload" %>
-  <%= javascript_importmap_tags "application-mcj" %>
+  <%= javascript_importmap_tags "application", importmap: MissionControl::Jobs.importmap  %>
```

Comment:
I believe passing a custom importmap here (https://github.com/rails/importmap-rails/pull/187) wasn't available from version < 2, perhaps with this change, we should require that version 🤔 Like: 
```diff
diff --git a/Gemfile.lock b/Gemfile.lock
index 185a970..5560422 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -29,7 +29,7 @@ PATH
   remote: .
   specs:
     mission_control-jobs (0.3.3)
-      importmap-rails
+      importmap-rails (~> 2.0)
       irb (~> 1.13)
       rails (>= 7.1)
       stimulus-rails
diff --git a/mission_control-jobs.gemspec b/mission_control-jobs.gemspec
index fc03152..07240ba 100644
--- a/mission_control-jobs.gemspec
+++ b/mission_control-jobs.gemspec
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   end

   spec.add_dependency "rails", ">= 7.1"
-  spec.add_dependency "importmap-rails"
+  spec.add_dependency "importmap-rails", "~> 2.0"
   spec.add_dependency "turbo-rails"
   spec.add_dependency "stimulus-rails"
   spec.add_dependency "irb", "~> 1.13"
```

Reply:
No, sorry, I don't know how I looked this up 😅  It was added [in version 1.2.1](https://github.com/rails/importmap-rails/releases/tag/v1.2.1), which is kind of old, but perhaps it's worth making that explicit for older apps 🤔 
```diff
diff --git a/mission_control-jobs.gemspec b/mission_control-jobs.gemspec
index fc03152..76e081e 100644
--- a/mission_control-jobs.gemspec
+++ b/mission_control-jobs.gemspec
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   end

   spec.add_dependency "rails", ">= 7.1"
-  spec.add_dependency "importmap-rails"
+  spec.add_dependency "importmap-rails", ">= 1.2.1"
   spec.add_dependency "turbo-rails"
   spec.add_dependency "stimulus-rails"
   spec.add_dependency "irb", "~> 1.13"
```